src/Utilities/StdArrayHelpers.hpp

```diff
@@ -166,6 +166,31 @@ decltype(auto) magnitude(const std::array<T, 3>& a) noexcept {
 }
 //@}
 
+/// \ingroup UtilitiesGroup
```

Comment:
If you want the comment to apply to all 3 overloads you can use Doxygen's
```cpp
// @{
/// \ingroup...
...
}
// @}
```
syntax

It would probably be good to note explicitly that the return type may end up being an expression template.

Reply:
I think this would be good to fix @nikwit :)